### PR TITLE
Display and style past dates on date picker

### PR DESCRIPTION
**Pull request by nathanstitt**
Just as I was starting to fork the component in order to add something like this, I discovered that they've just added a new `dayComponent` prop that allows you to render the dates yourself.

screen shot 2015-06-10 at 10 05 01 am

### CODE REVIEW

**Comment by nathanstitt** [comment]
This requires version 2.7 of react-widgets. 
I've bumped the version requirement in package.json, but everyone will have to make sure that they `nom install` after pulling this change.

File: src/components/tutor-input.cjsx [hunk 1]
```diff
@@ -28,6 +31,19 @@ TutorInput = React.createClass
...
+  propTypes:
+    date:  React.PropTypes.string.isRequired
+    label: React.PropTypes.string.isRequired
+
+
+  render: ->
+    className = if moment(@props.date).startOf('day') < TimeStore.getNow()
+      'invalid'
+    else
+      'valid'
```
Comment (philschatz) [inline src/components/tutor-input.cjsx]: Can you prefix this with `is-` (or the `invalid` case) and remove the other class name? I think that would also require adding a component name on it for the less file

**Comment by philschatz** [comment]
Howdy! Can you merge in changes from master?

**Comment by philschatz** [comment]
I am not seeing that behavior in the date picker. Also, previous dates seem to be selectable now:

image

**Comment by nathanstitt** [comment]
Hm.  It looks fine to me on my system,  I even manually deleted dist/* to force a complete style rebuild and cleared cache.  It's also strange that your picker isn't the same width as the input.  Did you `npm install` ?

**Comment by philschatz** [comment]
ah, you're right. I failed to read the instructions

**Comment by philschatz** [comment]
Hmm, so visually the dates are there but they are still clickable (and select Today). Is this a limitation of the `react-widget`

**Comment by nathanstitt** [comment]
Hmm good point. The selectable bit used to be a  limitation.   Anytime a day is clicked it closes the drop down.    We would then validate the selection and set it to today for past due selections. 

Now that we draw the day ourselves I bet we can intercept the click and call stopPropogation on the event.  Playing with that now

**Pull request merged**